src/cli.js

```diff
@@ -60,7 +61,17 @@ async function validateFile(filename, config, cache) {
     );
 
     const strictMode = config.verbose >= 2 ? "log" : false;
-    const { valid, errors } = await validate(data, schema, strictMode, cache);
+    const resolver = isUrl(schemaLocation)
```

Comment:
TODO: Do I also need to look at `$id` here (to cover the case where the schema itself is local, but `$id` is remote), or does ajv's resolution logic cover this case for me?

Reply:
This is fine - `$id` already takes precedence